src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -26,30 +26,36 @@
  *
  * <p>
- * This tool calculates the posterior genotype probability for each sample genotype in a VCF of input variant calls,
- * based on the genotype likelihoods from the samples themselves and, optionally, from input VCFs describing allele
- * frequencies in related populations. The input variants must possess genotype likelihoods generated by
- * HaplotypeCaller, UnifiedGenotyper or another source that provides <b>unbiased</b> genotype likelihoods.</p>
+ * The tool calculates the posterior genotype probability for each sample genotype in a given VCF format callset.
+ * The input variants must present genotype likelihoods generated by HaplotypeCaller, UnifiedGenotyper or other
+ * source that provides unbiased genotype likelihoods.</p>
  *
- * <h4>Statistical notes</h4>
- * <p>The AF field is not used in the calculation as it does not provide a way to estimate the confidence
- * interval or uncertainty around the allele frequency, unlike AN which does provide this necessary information. This
- * uncertainty is modeled by a Dirichlet distribution: that is, the frequency is known up to a Dirichlet distribution
- * with parameters AC1+q,AC2+q,...,(AN-AC1-AC2-...)+q, where "q" is the global frequency prior (typically q << 1). The
- * genotype priors applied then follow a Dirichlet-Multinomial distribution, where 2 alleles per sample are drawn
- * independently. This assumption of independent draws follows from the assumption of Hardy-Weinberg equilibrium (HWE).
- * Thus, HWE is imposed on the likelihoods as a result of CalculateGenotypePosteriors.</p>
+ * <p>
+ * The tool uses priors from three different data sources: (i) one or more supporting germline population callsets
+ * with specific annotation(s) if supplied, (ii) the pedigree for a trio if supplied and if the trio is represented
+ * in the callset under refinement, and/or (iii) the allele counts of the callset samples themselves given at least
+ * ten samples. It is possible to deactivate the contribution of the callset samples with the --ignore-input-samples
+ * flag.
+ * </p>
+ *
+ * <p>
+ * For more background information and for mathematical details, see GATK forum article at
+ * https://software.broadinstitute.org/gatk/documentation/article?id=11074.
+ * Additional GATK mathematical notes are presented as whitepapers in the <i>gatk</i> GitHub repository docs section
+ * at https://github.com/broadinstitute/gatk/tree/master/docs.
+ * </p>
  *
  * <h3>Inputs</h3>
  * <p>
  *     <ul>
  *         <li>A VCF with genotype likelihoods, and optionally genotypes, AC/AN fields, or MLEAC/AN fields.</li>
- *         <li>(Optional) A PED pedigree file containing the description of the relationships between individuals.</li>
+ *         <li>(Optional) A PED pedigree file containing the description of the relationships between individuals. Only
+ *         trio groups are considered in the calculations.</li>
```

Comment:
Should we be absolutely specific that we mean mother-father-offspring trios?

Reply:
What is meant by trios is I think clear. However, I see now we do not mention this anywhere and so I will add this.